src/test/java/com/woowacamp/soolsool/acceptance/AcceptanceTest.java

```diff
@@ -0,0 +1,22 @@
+package com.woowacamp.soolsool.acceptance;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
+class AcceptanceTest {
```

Comment:
```suggestion
public abstract class AcceptanceTest {
```

SonarLint 수정 사항 반영해주세요